WanderLost/WanderLost/Client/Components/Overlay.razor.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+using WanderLost.Shared.Data;
+using WanderLost.Client.Services;
+
+namespace WanderLost.Client.Components;
+
+public partial class Overlay : IOverlay
+{
+  [Inject]
+  private MerchantHubClient HubClient { get; set; }
+
+  [Inject]
+  private IJSRuntime JSRuntime { get; set; }
+
+  [Parameter]
+  public RenderFragment? ChildContent { get; set; }
+
+  public ActiveMerchantGroup? MerchantGroup;
+  public Dictionary<Guid, VoteType>? Votes;
+  public string? Server;
+  public string? Zone;
+
+
+  public bool isActive
+  {
+    get
+    {
+      return (this.MerchantGroup is not null) && (this.Server is not null) || (this.Zone is not null);
```

Comment:
```suggestion
      return ((this.MerchantGroup is not null) && (this.Server is not null)) || (this.Zone is not null);
```

I am too dumb to resolve predicate order, parenthesis would be really helpful